src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ name }) => (
+  <button type="button">{ name }</button>
+);
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: null,
+};
```

Comment:
- [ ] OPTIONAL - Since the Button component is always passed the `name` prop consider removing the defaultProps assignment and instead make the propType required.